### PR TITLE
(DOCSP-14671) Connect to MongoDB Realm Backend App

### PR DESCRIPTION
**Pull request by mongomoe**
### Issue JIRA link:
https://jira.mongodb.org/browse/DOCSP-14671

### Docs staging link (requires sign-in on MongoDB Corp SSO):
* Node: https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/DOCSP-14671-Connect-to-MongoDB-Realm-Backend-App/sdk/node/examples/connect-to-mongodb-realm-backend-app/
* Android: https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/DOCSP-14671-Connect-to-MongoDB-Realm-Backend-App/sdk/android/examples/connect-to-mongodb-realm-backend-app/
* .NET: https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/DOCSP-14671-Connect-to-MongoDB-Realm-Backend-App/sdk/dotnet/examples/connect-to-mongodb-realm-backend-app/

### CODE REVIEW

File: source/sdk/android/examples/connect-to-mongodb-realm-backend-app.txt [hunk 1]
```diff
@@ -1,8 +1,8 @@
-.. _android-init-appclient:
```
Comment (nathan-contino-mongo) [inline source/sdk/android/examples/connect-to-mongodb-realm-backend-app.txt:1]: Can we leave this ref here? It's fine to add the new one too. I'm almost certain that this ref is used in a few places across the android SDK doc, so even with your build link and proof that you haven't broken any links, I'm slightly hesitant to just remove it.

**Review by nathan-contino-mongo — APPROVED** [review]
LGTM

File: source/sdk/android/examples/connect-to-mongodb-realm-backend-app.txt (hunk 1 above)
Comment (mongomoe) [reply source/sdk/android/examples/connect-to-mongodb-realm-backend-app.txt:1]: Sure, totally understandable. I couldn't personally find uses of 'android-init-appclient' via a search through the repo, but I've added it back now for peace of mind :)

**Comment by cbush** [comment]
Hey @mongomoe @nathan-contino-mongo, this page is a cloud-specific page. The whole point of this effort was moving cloud-specific stuff towards the bottom of the TOC. Could we please be more careful next time? Thanks!

**Pull request merged**